inc/container.class.php

```diff
@@ -106,6 +106,12 @@ static function install(Migration $migration, $version) {
             foreach (json_decode($container['itemtypes']) as $itemtype) {
                $old_table = getTableForItemType(self::getClassname($itemtype, $old_name));
                $new_table = getTableForItemType(self::getClassname($itemtype, $container['name']));
+
+               //if table name more than 64 char extract only 64 char from it
+               if (strlen($new_table) > 64) {
+                  $new_table = substr($new_table, 0, 64);
```

Comment:
I think it will break usages of `getTableForItemType()` in different parts of GLPI. Classname should be truncated too.

Reply:
done, rename itemtype and tablename if needed

Reply:
Did you forget to push the commit ?